requirements/base.txt

```diff
@@ -40,9 +40,9 @@ click-plugins==1.1.1
     # via celery
 click-repl==0.3.0
     # via celery
-code-annotations==1.8.0
+code-annotations==2.1.0
     # via edx-toggles
-django==4.2.14
+django==4.2.17
```

Comment:
edx-platform is on `4.2.18`, and there's a few more diffs..

Could you run make upgrade again?